WhisperAPI/WhisperAPI/Services/Suggestions/SuggestionsService.cs

```diff
@@ -212,9 +217,15 @@ private IEnumerable<Document> FilterDocuments(ConversationContext conversationCo
             return documentsFiltered;
         }
 
-        private IEnumerable<QuestionToClient> GenerateQuestions(ConversationContext conversationContext, List<Document> documents)
+        private IEnumerable<QuestionToClient> GenerateQuestions(ConversationContext conversationContext, SearchQuery query)
         {
-            var questions = this.GetQuestionsFromDocument(conversationContext, documents).ToList();
```

Comment:
Did you mean to remove "GetQuestionsFromDocument()"? If yes, then you can delete the method completely, as it's no longer used elsewhere.